string folder_name
---
bool success
string message